src/parser/data/Variable.h

```diff
@@ -13,9 +13,9 @@
 #include "index/Index.h"
 
 class Variable {
+ public:
   std::string _name;
 
- public:
   explicit Variable(std::string name) : _name{std::move(name)} {

```

Comment:
Also note that I had to make the member public to be able to pass around `string*` which is done in the engine.